### PR TITLE
Türkçe translation update

### PR DESCRIPTION
**Pull request by umitseyhan75**

### CODE REVIEW

**Comment by jsixface** [comment]
@umitseyhan75 can you fix the failing build ?

**Comment by umitseyhan75** [comment]
@jsixface I have no idea why it is happening, I can try re-fork and then create a new pull request from it if you have do not have another idea.

File: app/src/main/res/values-tr/strings.xml [hunk 1]
```diff
@@ -35,72 +35,72 @@
...
     <string name="preferences_currently_not_syncing">Şu anda senkronize edilmiyor</string>
-    <string name="preferences_cloud_service">Senkronize hizmeti</string>
-    <string name="preferences_cloud_service_disabled">Kapalı</string>
+    <string name="preferences_cloud_service">Senkronizasyon hizmeti</string>
+    <string name="preferences_cloud_service_disabled">Devre dışı</string>
     <string name="preferences_cloud_service_nextcloud">Nextcloud</string>
-    <string name="preferences_sync_experimental_warning">Senkronize şu anda tamamlanmamış.\nHatalarla karşı karşıya gelebilirsin.</string>
-    <string name="preferences_nextcloud_limited_features">Nextcloud Notları etiket, eklenti, hatırlatıcı, görev listesi vb. özellikleri desteklemiyor.</string>
+    <string name="preferences_sync_experimental_warning">Senkronizasyon işlevi şimdilik deneyseldir.\nHatalarla karşılaşabilirsiniz.</string>
+    <string name="preferences_nextcloud_limited_features">Lütfen Nextcloud Notes'un etiketler, ekler, hatırlatıcılar, görevler gibi özellikleri desteklemediğini bilerek ilerleyin.</string>
```
Comment (jsixface) [inline app/src/main/res/values-tr/strings.xml:79]: I think it's failing because of the un-escaped `'` in `Notes'un`

**Review by jsixface — CHANGES_REQUESTED** [review]

File: app/src/main/res/values-tr/strings.xml (hunk 1 above)
Comment (umitseyhan75) [reply app/src/main/res/values-tr/strings.xml]: I can not believe Github thought that it needed to be escaped.

**Review by jsixface — APPROVED** [review]

**Pull request merged**